Utils/hdinsight-node-common/resources/htmlResources/hdinsight/job/html/common.js

```diff
@@ -52,19 +69,21 @@ function getMessageAsync(url, type, callback) {
                 if (s === '') {
                     return;
                 }
-                callback(s);
+                if (callback !== undefined) {
+                    callback(s);
```

Comment:
A good JS callback practice is the first argument is `err`, and the parameter starts from the second argument.